Official-Free-Code-Camp-Chat-Rooms.md

```diff
@@ -29,6 +29,6 @@ The following are our official chat rooms.
 | [CurriculumDevelopment](https://gitter.im/freecodecamp/CurriculumDevelopment) | help us improve our open source curriculum |
 | [DataScience](https://gitter.im/freecodecamp/DataScience) | help us understand our gigs and gigs of public data |
 
-By joining these chat rooms, you accept our [Code of Conduct](https://github.com/FreeCodeCamp/freecodecamp/wiki/Code-of-Conduct).
+By joining these chat rooms, you accept our [Code of Conduct](Code-of-Conduct).
```

Comment:
I think we should just keep the one on the site to avoid synchronisation issues.